escalate/core/templates/core/generic/list.html

```diff
@@ -13,18 +13,28 @@
 
 {% block content %}
 <div class="container">
-
+  <form method="post">{% csrf_token %}
     <table class="table table-bordered table-hover">
         <thead>
             <tr>
                 {% for header in table_columns %}
-                <th>{{ header }}</th>
+                <th class="text-nowrap">
+                  {{ header }}
+                  {% if header != "Actions" %}
+                    <button type="submit" name="sort" value="{{header}}_asc" onclick="this.form.submit()">
+                      <img src="{% static 'green_up_arrow.jpeg' %}" style="height:15px">
```

Comment:
I'll accept this, but in your next push you don't have to use an external jpeg for the arrows, you can use the already existing 'font-awesome' library in the HTML file. 

The buttons on the right for view, edit, delete buttons e.g. `<i class="fa fa-eye">`
You can check out all the "Directional Icons" section here: https://fontawesome.com/v4.7.0/icons/
It's also possible to change to color of the icons so you don't need images.

Another note: Currently we are using font awesome 4.7 but v5.x is available if you want you can try to upgrade that as well